lib/getLatestRelease.js

```diff
@@ -26,7 +26,7 @@ const getOS = str => {
     return 'Windows';
   } else if (
     includes(str, 'linux') ||
-    includes(str, 'appimage') ||
+    includes(str, '.AppImage') ||
```

Comment:
Does this still account for zipped AppImages? Otherwise, looks good.

Reply:
Yeah, the zipped AppImages have the "linux" marker, whereas the AppImages do not anymore